.travis.yml

```diff
@@ -18,9 +18,9 @@ matrix:
     addons:
       sauce_connect: true
       hosts:
-        - airtap.local
+      - airtap.local
 script: npm run $CMD
 env:
   global:
-  - secure: rE2Vvo7vnjabYNULNyLFxOyt98BoJexDqsiOnfiD6kLYYsiQGfr/sbZkPMOFm9qfQG7pjqx+zZWZjGSswhTt+626C0t/njXqug7Yps4c3dFblzGfreQHp7wNX5TFsvrxd6dAowVasMp61sJcRnB2w8cUzoe3RAYUDHyiHktwqMc=
-  - secure: g9YINaKAdMatsJ28G9jCGbSaguXCyxSTy+pBO6Ch0Cf57ZLOTka3HqDj8p3nV28LUIHZ3ut5WO43CeYKwt4AUtLpBS3a0dndHdY6D83uY6b2qh5hXlrcbeQTq2cvw2y95F7hm4D1kwrgZ7ViqaKggRcEupAL69YbJnxeUDKWEdI=
+  - secure: nk/kDUZV385ey9kkrGi6EHvnslTAW+GEHGrjbFIsb8Foq01bL9oj0uPaVc5kaVWCHst3WsIdVMA1Hos/lN/WMHRyfOiSrVVWv0sVnzg0YaM9cBbn5jwXn30Z645REgQcoEDKrYFp8rE3RpPFgYDFWNh948qwPtwtnwNhoq/Gd9A=
```

Comment:
@mcollina These are different from https://github.com/nodejs/readable-stream/pull/340/commits/40bd7adc039db9e494648c4200ed5cb6973f4c1e which worked?

Reply:
Oh, you might need to escape the `/` there

![screenshot from 2018-07-05 12-16-11](https://user-images.githubusercontent.com/308049/42317693-3d68c1d6-804d-11e8-8e6f-b770223a8ca8.png)